packages/@aws-cdk/toolkit-lib/lib/api/notices/web-data-source.ts

```diff
@@ -37,13 +47,29 @@ export class WebsiteNoticeDataSource implements NoticeDataSource {
   public readonly url: any;
 
   private readonly agent?: https.Agent;
+  private readonly skipNetworkCache?: boolean;
 
   constructor(private readonly ioHelper: IoHelper, props: WebsiteNoticeDataSourceProps = {}) {
     this.agent = props.agent;
     this.url = props.url ?? 'https://cli.cdk.dev-tools.aws.dev/notices.json';
+    this.skipNetworkCache = props.skipNetworkCache;
   }
 
   async fetch(): Promise<Notice[]> {
+    if (!this.skipNetworkCache) {
```

Comment:
why is this only for not skip network cache?

If you move this after the network check, it will always message. But do we even need this?